cmd/openshift-install/testdata/agent/image/validations/sno_invalid_platform.txt

```diff
@@ -0,0 +1,41 @@
+! exec openshift-install agent create image --dir $WORK
+
+stderr 'Only platform none supports 1 ControlPlane and 0 Compute nodes'
```

Comment:
This needs to be fixed and the agent installer should allow SNO in case of external platform as well , similar to assisted installer